src/serac/physics/state/finite_element_state.hpp

```diff
@@ -47,12 +47,84 @@ inline bool is_vector_valued(const GeneralCoefficient& coef)
  */
 class FiniteElementState : public FiniteElementVector {
 public:
+  using mfem::Vector::Print;
+
   /**
-   * @brief Use the finite element vector constructors
+   * @brief Main constructor for building a new finite element state
+   * @param[in] mesh The problem mesh (object does not take ownership)
+   * @param[in] options The options specified, namely those relating to the order of the problem,
+   * the dimension of the FESpace, the type of FEColl, the DOF ordering that should be used,
+   * and the name of the field
    */
-  using FiniteElementVector::FiniteElementVector;
-  using FiniteElementVector::operator=;
-  using mfem::Vector::Print;
+  FiniteElementState(
```

Comment:
The assignment operators were not actually getting called correctly, so I had to manually instantiate them. Due to compiler complaints, this meant I also had to manually instantiate the other constructors.